extract/html-table/src/parser.ts

```
@@ -0,0 +1,154 @@
+  rows.forEach((row, rowIndex) => {
+    Object.keys(rowspanCells).forEach((colIndex) => {
+      const colIndexNum = parseInt(colIndex, 10)
+      if (rowspanCells[colIndexNum].rowspan > 0) {
+        row.splice(colIndexNum, 0, rowspanCells[colIndexNum].value)
+        rowspanCells[colIndexNum].rowspan--
+      }
+    })
+
+    row.forEach((cell, colIndex) => {
+      const cellElement = parse(`<td>${cell}</td>`).querySelector('td')
+      if (cellElement && cellElement.getAttribute('rowspan')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)